polyfills/Map/polyfill.js

```diff
@@ -0,0 +1,124 @@
+(function(global) {
+
+
+	// Deleted map items mess with iterator pointers, so rather than removing them mark them as deleted. Can't use undefined or null since those both valid keys so use a private symbol.
+	var undefMarker = {};
```

Comment:
As these are in a closure, the symbols wouldn't be surfaced via `getOwnPropertySymbols()`.